src/types/index.d.ts

```diff
@@ -14,8 +14,10 @@ import Client from './client';
 import DefaultRequestExecutor from './default-request-executor';
 import RequestExecutor from './request-executor';
 
-export {
-  Client,
-  DefaultRequestExecutor,
-  RequestExecutor,
+declare const _default : {
+  Client: typeof Client,
+  DefaultRequestExecutor: typeof DefaultRequestExecutor,
+  RequestExecutor: typeof RequestExecutor,
 };
+
+export = _default;
```

Comment:
I think we want to use ES named exports in this file?

Reply:
I think you want to just do `export * from './client'`.  client.d.ts should also use named exports

Reply:
re-export seems to be working fine for both `import sdk = require('@okta/okta-sdk-nodejs')` and `import * as sdk from '@okta/okta-sdk-nodejs'`

Reply:
Both of those syntaxes are for compatibility with commonJS. In each case the module has a SINGLE export object ("sdk").

We'd like to do something like

`import { Client, RequestExecutor } from '@okta/okta-sdk-nodejs/types' (when we convert to ES then we will no longer need to add /types to the end)

Reply:
Yes, we got that. For types that are not re-exported through types index, a complete path should be specified (`import '@okta/sdk-node-js/src/types/http-error';`)